tests/ManageCourses.Tests/SmokeTests/BackendInternalApiEndpointTests.cs

```diff
@@ -0,0 +1,121 @@
+using FluentAssertions;
+using GovUk.Education.ManageCourses.Api.Model;
+using GovUk.Education.ManageCourses.ApiClient;
+using GovUk.Education.ManageCourses.Tests.UnitTesting.Client;
+using GovUk.Education.ManageCourses.Tests.TestUtilities;
+using GovUk.Education.ManageCourses.UcasCourseImporter;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Web;
+using Newtonsoft.Json;
+using System.Text;
+
+
+namespace GovUk.Education.ManageCourses.Tests.SmokeTests
+{
+    /// <summary>
+    /// These tests poke the http endpoints of the api whilst the api is running in a
+    /// a captive web host.
+    /// </summary>
+    [TestFixture]
+    [Category("Smoke")]
+    [Explicit]
+    public partial class BackendInternalApiEndpointTests : ApiSmokeTestBase
+    {
+        private const string Email = "feddie.krueger@example.org";
```

Comment:
spooky

Reply:
sourced from the `ApiEndpointTests` stems from all the nightmare we all have